doc/classes/Tween.xml

```diff
@@ -18,6 +18,7 @@
 		Many of the methods accept [code]trans_type[/code] and [code]ease_type[/code]. The first accepts an [enum TransitionType] constant, and refers to the way the timing of the animation is handled (see [url=https://easings.net/]easings.net[/url] for some examples). The second accepts an [enum EaseType] constant, and controls where the [code]trans_type[/code] is applied to the interpolation (in the beginning, the end, or both). If you don't know which transition and easing to pick, you can try different [enum TransitionType] constants with [constant EASE_IN_OUT], and use the one that looks best.
 		[url=https://raw.githubusercontent.com/godotengine/godot-docs/master/img/tween_cheatsheet.png]Tween easing and transition types cheatsheet[/url]
 		[b]Note:[/b] Tween methods will return [code]false[/code] if the requested operation cannot be completed.
+		[b]Note:[/b] To create a tween without instantiating a node, see [SceneTreeTween].
```

Comment:
```suggestion
		For an alternative method of tweening, that doesn't require using nodes, see [SceneTreeTween].
```
More accurate.

Reply:
Should this be moved before the "Note" above? (Since it does not have "Note:" now.)

Reply:
You can move it, I don't know if we have rules for that.

Reply:
No strong opinion from my end, it's true that normally `Note:`s come at the end so either it should keep a `Note:`, or it should be moved before the first `Note:`. Both are fine here IMO.

Reply:
Kept the `Note:` so it's similar to how `Timer` documentation handles `SceneTreeTimer`.